lib/siwapp/customers/customer.ex

```diff
@@ -66,4 +71,11 @@ defmodule Siwapp.Customers.Customer do
       add_error(changeset, hd(fields), "Either name or identification are required")
     end
   end
+
+  defp create_hash_id(changeset) do
+    name = get_field(changeset, :name)
+    identification = get_field(changeset, :identification)
+    hash = :crypto.hash(:md5, "#{name}#{identification}") |> Base.encode16()
+    change(changeset, %{hash_id: hash})
+  end
```

Comment:
I think you should use put_change instead. But read the docs of both carefully and decide.